src/utils/components/AddBootableVolumeModal/utils/constants.ts

```diff
@@ -84,11 +84,13 @@ export const emptySourceDataVolume: V1beta1DataVolume = {
   },
   spec: {
     storage: {
+      accessModes: [],
       resources: {
         requests: {
           storage: '',
         },
       },
+      volumeMode: '',
```

Comment:
if the storage profile settings are applied, will accessModes and volumeMode appear as empty in the YAML?

Reply:
They were empty in the YAML regardless. I removed these lines because they didn't appear to make any difference at all.